### ISSUE TITLE
ImageNet ckpt

### ISSUE THREAD

**Issue by Facebear-ljx** [swyoon/Diffusion-by-MaxEntIRL#1]
Hi! Thanks for your wonderful work. I am wondering when will you release the ImageNet 64x64 dmxi ckpt

**Comment by swyoon** [comment]
Hi, thanks for filing an issue. Could you check the dropbox link once again and see if you can download imagenet checkpoints? It was my mistake only uploading cifar10 checkpoints.

**Comment by Facebear-ljx** [comment]
Hi! Thanks for providing the ckpt! I firstly tried to reproduce the CIFAR-10 results by running your provided command (with a few modification on GPU and proc numbers):

`$ CUDA_VISIBLE_DEVICES=0,1 torchrun --nproc_per_node=2 train_cifar10.py \
    --config configs/cifar10/T10.yaml --dataset configs/cifar10/cifar10.yaml`

However, the best FID I got is around 5.X, not 3.X as reported in your paper. I planned to develop one project based on your codebase. So, it would be really appreaciated if you could help me check what's wrong here.

These are my running results after trying several different seeds:

https://wandb.ai/facebear/dxmi_cifar10_T10

**Comment by swyoon** [comment]
Hi, 

If you didn't modify the provided training script, the FID score logged during training is just a proxy metric computed with only 10,000 (CIFAR10) or 5,000 (imagenet) samples. 
This proxy metric tends to be larger than the "proper" metric that is computed using 50,000 samples. I think the proxy FID of 5 is within the expected range.
The final FID reported in the manuscript is computed using 50,000 samples, using separate scripts such as `generate_cifar10.py` or `generate_large.py`.

**Comment by Facebear-ljx** [comment]
Thank you so much!
